### ISSUE TITLE
Allow to disable zoom buttons

### ISSUE THREAD

**Issue by TimMagee** [osmandapp/Osmand#4117]
Currently I cannot find a way to disable the zoom buttons. However, zoom can be accomplished (much more easily for me) with the pinch gesture.

I propose that we allow the user to disable the zoom buttons just as we allow the user to disable the 'quick action' button. This will make the interface look less cluttered and will make the app more user friendly for new users who expect a clean interface.

**Comment by varlamov** [comment]
I totally agree with that proposal. It's nice to have zoom buttons on by default for accessibility reasons but for users that do use gestures to zoom those buttons are useless and occupies screen real estate. I'd proposed to keep them on by default bot allow to hide them in the menu

**Comment by llbstr** [comment]
I would love to have that option too :)

**Comment by duiveldoder** [comment]
Yes! Thumbs up for this proposal - we need to have the ability to hide these.

**Comment by Salamimonster** [comment]
I have to say: On a touch screen based device, the buttons are ~~prehistoric~~ out of date and a waste of screen space.
I read somewhere here, that buttons are more save to operate during driving. That is not truth - it needs mutch more attention to hit a 10x10mm button, than to simply tab somewhere on a 6" screen with two fingers and to pinch and zoom.

**Comment by DmitryAlexei** [comment]
To disable zoom buttons you need to enable Full screen mode in profile settings:
- go to menu > settings > your profile > general settings 
- swipe to the bottom of the settings list and enable Full screen mode

When that option enabled, you need to short tap an empty place on the map and all the buttons disappears.

**Comment by Salamimonster** [comment]
Thank you for the quick response, Dmitry. But "full screen mode" is **_not_** the same, as being able to disable the ~~windows XP style~~ zoom buttons.

Feel free to make a polling, to find out what the users want. Or (even better) code a counter in the app and count the times, that users use pinch to zoom vs. the times users use ~~floppy drive technique~~ zoom buttons.

**Comment by dmpr0** [comment]
Done as part of https://github.com/osmandapp/OsmAnd/issues/21815
